mods/tuxemon/db/technique/frostbite.json

```diff
@@ -4,6 +4,7 @@
   "animation": "lance_ice",
   "category": "physical",
   "effects": [
+    "status status_poison",
```

Comment:
Seems odd...

Reply:
I'm testing with one monster with Frostbite.
Do you want to get rid of status.py and handle the poisoning (everything else included) inside specific py files (like poison.py)?
If yes, then I'm in.

Reply:
It works better